src/app/armory/weapon-roll-export.test.ts

```diff
@@ -0,0 +1,121 @@
+import { DimItem } from 'app/inventory/item-types';
+import { DestinySocketCategoryStyle } from 'bungie-api-ts/destiny2';
+import { formatWeaponRollForExport } from './weapon-roll-export';
+
+// Mock socket utils
+jest.mock('app/utils/socket-utils', () => ({
+  getWeaponArchetypeSocket: jest.fn(),
+  getSocketsWithStyle: jest.fn(),
+  isWeaponMasterworkSocket: jest.fn(),
+}));
```

Comment:
I hate mocking. I would much rather you follow what other tests do, which is to load the real manifest using helpers in `src/testing/test-utils.ts` and then run the function on real items. Snapshot tests are also appropriate here.